src/main/java/de/craften/ui/swingmaterial/MaterialWindow.java

```diff
@@ -8,29 +8,216 @@
+     * see {@link #shadowPane}.
+     */
+    private Container contentPane;
+    /**
+     * This panel is used as background of the window itself, so it drops a
+     * shadow.
+     * @see ShadowPane
+     */
+    protected ShadowPane shadowPane;
+    
+    /**
+     * Default constructor for a {@link MaterialWindow}. A container for the
+     * shadow is deployed if and only if the current graphics evironment
+     * supports per-prixel translucency for top-level containers (windows in
+     * general).
+     */
     public MaterialWindow() {
-        GraphicsEnvironment ge = GraphicsEnvironment.getLocalGraphicsEnvironment();
-        if (ge.getDefaultScreenDevice().isWindowTranslucencySupported(GraphicsDevice.WindowTranslucency.PERPIXEL_TRANSLUCENT)) {
-            setUndecorated(true);
+        super();
+        setUndecorated(true);
+        if (Utils.isTranslucencySupported()) {
             setBackground(new Color(255, 255, 255, 0));
-            setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-            setContentPane(new ShadowPane());
-            getRootPane().putClientProperty("Window.shadow", Boolean.FALSE);
-            getRootPane().setWindowDecorationStyle(JRootPane.NONE);
-            addComponentListener(new ComponentAdapter() {
-                @Override
-                public void componentResized(ComponentEvent componentEvent) {
-                    setShape(new RoundRectangle2D.Float(0, 0, getWidth(), getHeight(), 3, 3));
-                }
-            });
+            //This default behavior should NOT be set here!
+            //setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+            super.setContentPane((shadowPane = new ShadowPane()));
+            contentPane = new JPanel();
+            shadowPane.add(contentPane);
+        } else {
+            contentPane = super.getContentPane();
         }
+        getRootPane().putClientProperty("Window.shadow", Boolean.FALSE);
+        getRootPane().setWindowDecorationStyle(JRootPane.NONE);
+        addComponentListener(new ComponentAdapter() {
+            @Override
+            public void componentResized(ComponentEvent componentEvent) {
+                setShape(new RoundRectangle2D.Float(0, 0, getWidth(), getHeight(), 3, 3));
+            }
+        });
+        light();
     }
 
-    private class ShadowPane extends JPanel {
+    @Override
+    public Container getContentPane() {
+        return contentPane;
+    }
+
+    @Override
+    public void setContentPane(Container contentPane) {
+        this.contentPane = contentPane;
+    }
+    
+    /**
+     * Chainable method to quickly get a {@link MaterialWindow} with a dark
+     * background and white foreground. You can append it to your constructor
+     * like this: {@code MaterialWindow myWindow = new MaterialWindow.dark()} or
+     * just use it alone to switch the foreground/background colors.
+     * @return this window
+     */
+    public MaterialWindow dark() {
```

Comment:
I'm not sure about these `dark()` and `light()` methods. Maybe we could add a `MaterialWindow.Style` enum with `LIGHT` and `DARK` values and add a `setStyle` method (a `getStyle` method would not really be needed for now). We could also add a constructor that takes a `style`.
What do you think?